### PR TITLE
[Editorial] Finding Borders

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by brebenelmihnea** [comment]
https://cses.fi/problemset/result/10360699/
image

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by brebenelmihnea** [comment]
The fastest approval :))

**Comment by SansPapyrus683** [comment]
OOPS DID NOT MEAN TO TURN ON AUTOMERGE
SORRY RYAN

File: content/6_Advanced/String_Search.mdx [hunk 1]
```diff
@@ -595,18 +595,6 @@ int main() {
 
 <Problems problems="aho" />
 
-<IncompleteSection>
+# Practice Problems
```
Comment (ryanchou-dev) [inline content/6_Advanced/String_Search.mdx:598]: are these problems for the entire module? 

also just leave it as problems, not practice problems for consistency. you can just fix this when u add the other problem :)

Comment (brebenelmihnea) [reply content/6_Advanced/String_Search.mdx:598]: Sure!

**Pull request merged**